docs/sql-guide/statements/statement-insert-bulk-orc-example.md

```diff
@@ -0,0 +1,61 @@
+---
+title: BULK INSERT ORC example
+layout: default
+parent: Statements
+grand_parent: SQL guide
+nav_order: 5
+---
+
+# Ingest a ORC file with BULK INSERT
+
+This example demonstrates how to:
+
+* Create a FeatureBase table with a required structure
+* Copy and transform data from an ORC source
+* Use the `BULK INSERT` statement to copy data from the source to the target table.
+
+## Before you begin
+
+{% include /sql-guide/bulk-insert-eg-before-begin.md%}
+
```

Comment:
Insert reference to what ORC is here:

* [Apache ORC file format](https://github.com/apache/orc){:target="_blank}

Reply:
i have included the library we referred - https://github.com/scritchley/orc

Reply:
have pointed users to the Go page instead of this as it's more complete and has much of the same information.